### PR TITLE
Update docstrings with plot examples

### PR DESCRIPTION
**Pull request by AlexMetsai**
I added the 'plot show' in about half of the examples. This could be done in a single PR, but I'm not sure if I'll finish the rest today, since I prefer running the examples on my computer to avoid possible mistakes like missing a figure in examples with more than one plot.
Today or tomorrow I'll create a pull request with all the rest (intentional rhyme).

#### What does this implement/fix? Explain your changes.
Includes the line to view the plot, in case the example code is read/run by the user.

#### Any other comments?
No.

### CODE REVIEW

**Comment by AlexMetsai** [comment]
Finished with the remaining examples.

**Comment by bmcfee** [comment]
Thanks @AlexMetsai !  Does this mean that PR #881 is redundant?

In general, I'd prefer to keep this all in one PR.  You can add more commits to this one later, and let me know when it's ready for review.

**Comment by AlexMetsai** [comment]
Yes, it includes the previous PR's commit. I hadn't noticed that it wasn't merged, you want me to close it, or is it you that should do it?

I think the PR is ready for review, I hope I haven't missed or messed anything up!

**Comment by bmcfee** [comment]
@AlexMetsai as a general rule, it's bad to make multiple partial pull requests for the same bundle of changes.  This makes it harder for us to review, and partial merges leave the master branch in an incomplete state.  It really would be better if you could bring in all of the relevant commits to this PR before we merge.

Thanks!

**Comment by AlexMetsai** [comment]
Yes, I know, this PR already includes the change(s) from the previous one. 
#881 is redundant as you're saying, I just don't know if I should be the one to close that PR or you.

**TL;DR** This PR has everything you need, ignore/close/tell me to close the previous PR.

P.S. Sorry for opening two PRs in the first place, I understand the inconvenience. I did so 'cause I (mistakenly) though the previous one was already merged. I hope it's not a lot of trouble now that everything's in this PR.

Cheers and I'll keep that in mind! ^__^

**Comment by bmcfee** [comment]
Ok, thanks for clarifying!  Just let me know when this one's done / ready for CR.

**Comment by AlexMetsai** [comment]
It's ready!

**Comment by bmcfee** [comment]
:shipit: ! Thanks again @AlexMetsai !

**Comment by AlexMetsai** [comment]
You' re welcome!
